src/apis/avm/credentials.ts

```diff
@@ -25,12 +25,18 @@ export const SelectCredentialClass = (credid:number, ...args:Array<any>):Credent
 
 export class SECPCredential extends Credential {
   protected _typeName = "SECPCredential";
+  // TODO - Bump _typeID to *_CODECONE after Apricot
```

Comment:
I don't immediately see an easy way to do this but I may as well mention it, is there a way we could bump them all at once like how we're doing with `AVMConstants.LATESTCODEC`?

Reply:
At the moment I don't see the obvious way either but I'm happy to brainstorm on it with you. I think this solution works for the apricot release.